akka-projection-cassandra/src/test/scala/akka/projection/cassandra/CassandraOffsetStoreSpec.scala

```diff
@@ -6,27 +6,24 @@ package akka.projection.cassandra
 
 import java.util.UUID
 
-import scala.concurrent.ExecutionContext
-import scala.util.Try
-
-import akka.actor.testkit.typed.scaladsl.LogCapturing
-import akka.actor.testkit.typed.scaladsl.ScalaTestWithActorTestKit
-import akka.persistence.query.Sequence
-import akka.persistence.query.TimeBasedUUID
+import akka.actor.testkit.typed.scaladsl.{ LogCapturing, ScalaTestWithActorTestKit }
+import akka.persistence.query.{ Sequence, TimeBasedUUID }
 import akka.projection.ProjectionId
 import akka.projection.cassandra.internal.CassandraOffsetStore
 import akka.stream.alpakka.cassandra.scaladsl.CassandraSessionRegistry
 import org.scalatest.wordspec.AnyWordSpecLike
 
+import scala.concurrent.ExecutionContext
+import scala.util.Try
+
 class CassandraOffsetStoreSpec extends ScalaTestWithActorTestKit with AnyWordSpecLike with LogCapturing {
+  private implicit val ec: ExecutionContext = system.executionContext
 
   private val session = CassandraSessionRegistry(system).sessionFor("akka.projection.cassandra")
-  private implicit val ec: ExecutionContext = system.executionContext
-  private val offsetStore = new CassandraOffsetStore(session)
+  private val offsetStore: CassandraOffsetStore = new CassandraOffsetStore(session)
 
   override protected def beforeAll(): Unit = {
     super.beforeAll()
-    // reason for setSchemaMetadataEnabled is that it speed up tests
```

Comment:
?

Reply:
Collateral damage from earlier refactoring.  I reverted a bunch of files including this one.